api/client/transform/fortio.proto

```diff
@@ -42,6 +47,18 @@ message DurationHistogram {
   uint64 Count = 1;
 
   repeated DataEntry Data = 2;
+
+  double Min = 3;
+
+  double Max = 4;
+
+  double Sum = 5;
+
+  double Avg = 6;
+
+  double StdDev = 7;
+
+  repeated FortioPercentile Percentiles = 8;
```

Comment:
Does this message have to be prefixed with `Fortio`, since it's already in the `FortioResult` proto? I guess it makes it easier to use in the transform code, where you have Fortio and NH percentiles.

Reply:
Well, there’s already a ‘Percentile’ message defined in NH’s own output proto and using the same name here broke the build.